modules/spdlog/1.15.0.bcr.2/MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+module(
+    name = "spdlog",
+    version = "1.15.0.bcr.2",
+    compatibility_level = 1,
```

Comment:
Add `bazel_compatibility = [">=7.2.1"],` since you depend on overlay feature from Bazel